shared/haml/help_button.haml

```diff
@@ -8,7 +8,8 @@
 .help_button{class: "hide-mobile", id: "help-button"}
   .help_contents{style: 'display: none', id: "help-contents"}
     - help_items.each do |entry|
-      %a.linktag{id: entry[:id], href: entry[:url]}=entry[:title]
+      - target = entry[:target] ? entry[:target] : "_self"
+      %a.linktag{id: entry[:id], href: entry[:url], target: target}=entry[:title]
```

Comment:
I think we could also leave it as nil for a fallback and then no `target` field will be generated.